### ISSUE TITLE
Add focus state styles for the cookie banner

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#2128]
## What
Further to https://github.com/alphagov/govuk-frontend/issues/2098, we should add a focus style for the cookie banner. This could be similar to the notification banner focus state.

We could add this in https://github.com/alphagov/govuk-frontend/pull/2126 or separately if we don't want to block merging that PR.

## Why

The cookie banner needs to have a focus state as it should be focused after the user accepts or rejects cookies.

## Who needs to know about this
Designer, developer

## Done when

- [ ] We've decided what the focus state should be
- [ ] We've added the styles
- [ ] The styles have been reviewed by a designer

**Comment by CharlotteDowns** [comment]
Two ideas for team review:

### Offset outline
<img width="1439" alt="Cookie banner message focus state using an offset outline" src="https://user-images.githubusercontent.com/56260216/106461863-43777280-648d-11eb-8dca-c56d148a78d0.png">

### Background and underline to text 
<img width="1440" alt="Cookie banner message focus state to add yellow background to text and black underline" src="https://user-images.githubusercontent.com/56260216/106461860-42464580-648d-11eb-9eb8-62795917e9c1.png">

**Comment by vanitabarrett** [comment]
We have decided to publish the component without a visible focus state to start with. 

From @hannalaakso: The focused cookie banner is the first element on the page and the last thing the user
interacted with prior to it gaining focus. We therefore assume that moving focus to it is not going to surprise users, and that giving it a visible focus indicator could be more confusing than helpful, especially as the element is not normally keyboard operable.

We have flagged this in the research section of the guidance as something to monitor.

A related discussion: w3c/wcag#1001

It's also worth noting that this is different to what we do with the notification banner which does have a visible focus indicator. However, the notification banner is further down the page and the user might not have interacted with it prior to it gaining focus so it is helpful to indicate to users that the focus has moved to it (even though the element, like the cookie banner, isn’t normally keyboard operable).
